vite.config.js

```diff
@@ -59,11 +58,38 @@ export default defineConfig({
   },
   server: {
     watch: {
-      usePolling: true,   // Activa polling para detectar cambios en entornos donde el file watching puede fallar
```

Comment:
It might be useful to leave these comments so that a future developer knows how to change the Vite configurations.

Reply:
Agregue nuevamente el comentario